### DISCUSSION TITLE
file uploaded to s3 via sync not visible in browser

### DISCUSSION THREAD

**Discussion by everythingness** [iterate-ch/cyberduck#14904] [Q&A]
when i sync a folder to a bucket on s3, a new file created locally is uploaded but not visible in the browser.

**Comment by tatze96** [comment]
I can't reproduce the described issue on my side. I see that the Cyberduck browser needs a reload before displaying the newly synchronized items, but the files are visible within Cyberduck and the server's web interface.

Please share which synchronization mode (mirror, upload, download) and which version of Cyberduck was used.
